changelog.d/628.bugfix

```diff
@@ -0,0 +1 @@
+Make sure we use correct displaynames and avatars on Slack even if we lost track of room state updates
```

Comment:
Hm, this feels more like developer talk than something users might understand. Would prefer something like: "Improve reliability of Matrix users being bridged with correct displayname and avatar"

Reply:
Good point. Fixed in c921200